File: C/operadora.c
#include <stdio.h>

int main() {

    int minutos;
    double custo;

    printf("Digite a quantidade de minutos: ");
    scanf("%d", &minutos);

    if (minutos <= 100) {
        custo = 50.00;
    }
    else {
        custo = (minutos - 100) * 2 + 50.00;
    }

    printf("Valor a pagar: R$ %.2lf", custo);

    return 0;
}

File: Java/Operadora.java
import java.util.Locale;
import java.util.Scanner;

public class Operadora {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int minutos;
        double valor;

        System.out.print("Digite a quantidade de minutos: ");
        minutos = sc.nextInt();

        if (minutos <= 100) {
            valor = 50.00;
        }
        else {
            valor = (minutos - 100) * 2 + 50.00;
        }

        System.out.format("Valor a pagar : R$ %.2f", valor);

        sc.close();
    }
}